server.js

```diff
@@ -18,10 +18,62 @@ app.use(express.json());
 
 // Start defining your routes here
 app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+  res.send("Think happy thoughts, think happy thoughts");
 });
+//////////////////
+const ThoughtSchema = new mongoose.Schema({
+  message: {
+    type: String,
+    required: true,
+    minlength: 5,
+    maxlength: 140,
+  },
+  hearts: {
+    type: Number,
+    default: 0
+  },
+  createdAt: {
+    type: Date,
+    default: (() => new Date())()
```

Comment:
I believe it's something wrong with the time stamp. When posting a thoughts it says "posted 15 minutes ago" even if I posted it now. It doesn't matter if I reload the page. Your code differs a bit from mine:

`  default: () => new Date() `